OSC/ondemand
# continue support for project templates

This continues the work for #2720.

The easiest way to test this is to 
* configure `OOD_PROJECT_TEMPLATE_DIR` to point to some directory
* create a couple projects with descriptions titles and icons
* copy these projects you've just created to `OOD_PROJECT_TEMPLATE_DIR`
* Now you should see them as templates when you _press create from template_.



┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1135148780858012/1204375724018215) by [Unito](https://www.unito.io)
